packages/react-native/src/primitives/Label/Label.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { LabelProps } from './types';
+import { styles } from './styles';
+
+export default function Label({
+  children,
+  style,
+  ...rest
+}: LabelProps): JSX.Element {
+  return (
+    <Text {...rest} accessibilityRole="text" style={[styles.label, style]}>
```

Comment:
Let's make sure we `Omit` the `accessibilityRole` on the `LabelProps` since it's not configurable